src/desktop/components/artwork_metadata_stub/index.styl

```diff
@@ -1,10 +1,13 @@
 @require '../stylus_lib'
+@require '../../lib/palette'
+
+LINE_HEIGHT = 21px
```

Comment:
👍

Reply:
🙈  — is done to preserve the placeholder sizing, in this case